package.json

```diff
@@ -93,8 +93,8 @@
     "yorkie": "^2.0.0"
   },
   "dependencies": {
-    "@eslint/core": "^0.17.0",
-    "@eslint/plugin-kit": "^0.4.1",
+    "@eslint/core": "^1.0.0",
```

Comment:
Same question as https://github.com/eslint/json/pull/190#discussion_r2584390444. We can clarify there, just wanted to leave a note here as well so that we don't merge until this is clarified.